specification/ml/get_trained_models/MlGetTrainedModelRequest.ts

```diff
@@ -82,6 +82,10 @@ export interface Request extends RequestBase {
      * none. When supplied, only trained models that contain all the supplied
      * tags are returned.
      */
-    tags?: string
+    tags?: string | string[]
+    /**
+     * @since 7.17.0
```

Comment:
Looks like this is deprecated per https://github.com/elastic/elasticsearch/blob/master/rest-api-spec/src/main/resources/rest-api-spec/api/ml.get_trained_models.json#L46
```suggestion
     * @since 7.17.0
     * @deprecated
```

Reply:
The entire API and `include_model_definition` parameter were first added in 7.6: https://github.com/elastic/elasticsearch/pull/49052/

`include_model_definition` was later deprecated in 7.10 by https://github.com/elastic/elasticsearch/pull/62834

It looks like  theJSON spec was not correctly updated at the same time hence the confusion.